### ISSUE TITLE
C#.NET Windows Applications.

### ISSUE THREAD

**Issue by AliCharper** [GuillaumeSalles/redux.NET#71]
Hi,

Can we use this platform for C#.NET Windows Applications or no?.

**Comment by Vazerthon** [comment]
I've played around with it in a C# WPF app, it works well.

There seem to be some questions about the future of redux.NET though, you might want to have a look at [An Immutable Asynchronous State Holder in C# - blog post by Atomic](https://spin.atomicobject.com/2017/07/31/c-sharp-state-holder/) as an alternative.

**Comment by AliCharper** [comment]
so you mean I should forget it.

**Comment by Vazerthon** [comment]
I wouldn't got that far. The maintainer stated in #69 that they don't have time to maintain this project any more. Some others have suggested taking it over. I found it pretty stable for my needs but if you plan to go into production with a banking app you might want to look elsewhere.

**Comment by AliCharper** [comment]
so what should we do?. I need something like this.

**Comment by Vazerthon** [comment]
Redux is a pretty well documented concept, you could always build your own. Or just use this until you find a limitation then swap out to something else.

**Comment by sclarke81** [comment]
I think @Vazerthon has summarised where things are at pretty well. I created a project called TodoMVC.NET to try out some different frameworks. So there's a Redux.NET example on there. I like it and think it works well, but between using it myself and reading the discussions on here I think it can be improved.

As has been mentioned, the current owner isn't going to be able to support Redux.NET but they have offered to pass on control to a team that wants to take it forwards. If I decide to make use of Redux.NET then I'll volunteer to be on that team. I feel like this is the only way that I'd be able to dedicate the effort required and have the necessary insight. I'd suggest that you take the same kind of approach. On a positive note there's actually relatively little code so maintenance shouldn't be too difficult if you wanted to use it as is.

**Comment by jjwilliams42** [comment]
@sclarke81 I agree with your comment about there being relatively little code to maintain. The pattern it's following isn't changing, with a solid foundation in place, there really shouldn't be a whole lot of "new features", maybe some bug fixes here and there.

We are running into state problems in our WPF application and I'm considering picking this up.

**Comment by hoangdovan** [comment]
@AliCharper Yes, this is dead project and you should not use it anymore. @Vazerthon is right, Redux is quite simple. I also created my own version to use for my projects.
